resources/schema.py

```diff
@@ -572,6 +603,8 @@ def mutate_and_get_payload(cls, root, info, **input):
             raise VenepaikkaGraphQLError(e)
 
         update_object(harbor, input)
+        add_map_files(HarborMap, input.pop("add_map_files", []), harbor)
```

Comment:
I think it's better to do these before running `update_object`, so it would not try to do `set_attribute(harbor, "add_map_files", [HarborMap])` and so on.

Reply:
Good catch!